bin/alluxio

```diff
@@ -149,7 +149,7 @@ function runJavaClass {
               CLASS_ARGS+=("${arg}")
       esac
   done
-  "${JAVA}" -cp ${CLASSPATH} ${ALLUXIO_USER_JAVA_OPTS} ${ALLUXIO_SHELL_JAVA_OPTS} ${CLASS} ${PARAMETER} "${CLASS_ARGS[@]}"
+  exec "${JAVA}" -cp ${CLASSPATH} ${ALLUXIO_USER_JAVA_OPTS} ${ALLUXIO_SHELL_JAVA_OPTS} ${CLASS} ${PARAMETER} "${CLASS_ARGS[@]}"
```

Comment:
Oh I should make this a separate PR. I'll remove it.

I think technically it is a bug. Without using `exec` a `ctrl-c` doesn't actually kill the java process running if it's a CLI program. Rather it just kills the shell running the CLI command, so a haning CLI command i.e because of a large listStatus, or master unavailable doesn't actually get killed if the USER ctrl-Cs and it may continue to run in the background.

Reply:
reverted